.github/workflows/codecov-main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [14, 16, 18]
+        node-version: [16, 18, 20]
```

Comment:
**Suggestion:** Consider adding a 'fail-fast: false' strategy to your matrix. This will ensure that all jobs run to completion, even if one fails. This can be useful for identifying multiple issues across different Node.js versions at once.
```suggestion
    strategy:
      fail-fast: false
      matrix:
        node-version: [16, 18, 20]
```